### PR TITLE
fix: hugegraph-spark-loader shell string length limit

### LINKED ISSUE #460: [Bug] hugegraph-spark-loader脚本执行EXEC 命令时可能触发长度限制
### Bug Type (问题类型)

None

### Before submit

- [X] I had searched in the [issues](https://github.com/apache/hugegraph-toolchain/issues) and found no similar issues.

### Environment (环境信息)

- Server Version: v1.0.0
- Toolchain Version: v1.0.0
- Data Size:  xx vertices, xx edges 

### Expected & Actual behavior (期望与实际表现)

error message:
` --graph hugegraph --batch-size 1499 --host 10.136.29.143 --port 8080 --execute-date 2023-05-11 --sink-type false  --vertex-partitions 120 --edge-partitions 120 --graph-type entity --file /xxx/vertexs-struct.json: **cannot execute: File name too long**
`
hugegraph-spark-loader.sh:
....
echo "${CMD}"
exec "${CMD}"

The above error message will be reported when executing exec "${CMD}"，My solution is to remove the double quotes,See if the community has a better solution, thanks

### Vertex/Edge example (问题点 / 边数据举例)

_No response_

### Schema [VertexLabel, EdgeLabel, IndexLabel] (元数据结构)

_No response_

### PR DESCRIPTION
**Pull request by gaoyuan5251**
fix #460
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Comment by gaoyuan5251** [comment]
@imbajin hi bro, Do I need to fix this error check?

File: hugegraph-loader/assembly/static/bin/hugegraph-spark-loader.sh [hunk 1]
```diff
@@ -35,4 +35,4 @@ CMD="${SPARK_HOME}/bin/spark-submit
     --jars $(echo "${LIB_DIR}"/*.jar | tr ' ' ',') ${ASSEMBLY_JAR_NAME} ${HUGEGRAPH_PARAMS}"
 
 echo "${CMD}"
-exec "${CMD}"
+exec ${CMD}
```
Comment (imbajin) [inline hugegraph-loader/assembly/static/bin/hugegraph-spark-loader.sh:38]: another `SHELL TRAP` @zyxxoo

**Comment by imbajin** [comment]
> @imbajin hi bro, Do I need to fix this error check?

Nope If u mean this
image

**Review by imbajin — APPROVED** [review]

**Review by liuxiaocs7 — APPROVED** [review]
I've modified this before and it works

**Comment by liuxiaocs7** [comment]
Hi, @gaoyuan5251, you could refer [this](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) to link pr with associated issue, once the pr is merged the issue will be closed automatically.

**Comment by gaoyuan5251** [comment]
> 

Do I need to repush it?  Or just wait for pr to merge?

**Comment by gaoyuan5251** [comment]
Do I need to repush it?  Or just wait for pr to merge? @liuxiaocs7

**Review by simon824 — APPROVED** [review]

**Pull request merged**